<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LevelSetSegmentationView</class>
 <widget class="QWidget" name="LevelSetSegmentationView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>371</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>381</width>
     <height>341</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Image</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QmitkDataStorageComboBox" name="DataSelector"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Sigma</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="SigmaLE">
        <property name="text">
         <string>1.0</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Alpha</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="AlphaLE">
        <property name="text">
         <string>30</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Beta</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="BetaLE">
        <property name="text">
         <string>20</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>Stoping Value</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="StoppingValueLE">
        <property name="text">
         <string>150</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_5">
      <item>
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>Advection</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="AdvectionScalingLE">
        <property name="text">
         <string>1.0</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_7">
        <property name="text">
         <string>Curvature</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="CurvatureScalingLE">
        <property name="text">
         <string>1.0</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_8">
        <property name="text">
         <string>Propagation</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="PropagationScalingLE">
        <property name="text">
         <string>1.0</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_6">
      <item>
       <widget class="QLabel" name="label_9">
        <property name="text">
         <string>Number Of Interaction</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="NumberOfInteractionLE">
        <property name="text">
         <string>500</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QmitkPointListWidget" name="SeedWidget" native="true">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>100</height>
       </size>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="ApplyBtn">
      <property name="text">
       <string>Apply</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="StopBtn">
      <property name="text">
       <string>Stop</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QmitkDataStorageComboBox</class>
   <extends>QComboBox</extends>
   <header>QmitkDataStorageComboBox.h</header>
  </customwidget>
  <customwidget>
   <class>QmitkPointListWidget</class>
   <extends>QWidget</extends>
   <header>QmitkPointListWidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
